### PR TITLE
Added html and message count for stats page

### PR DESCRIPTION
**Pull request by linkaren**
Beginning steps for the Stats Page!

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -114,4 +115,11 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+  /** Returns the total number of messages for all users. */
+  public int getTotalMessageCount() {
+    Query query = new Query("Message");
+    PreparedQuery results = datastore.prepare(query);
+    return results.countEntities(FetchOptions.Builder.withLimit(1000));
```
Comment (FluffySheep) [inline src/main/java/com/google/codeu/data/Datastore.java]: Let's make the 1000 a static const int up at the top. We don't want magic numbers here.

Here's the reference:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

File: src/main/java/com/google/codeu/servlets/StatsPageServlet.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
+
+  /** Responds with site statistics in JSON. */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
```
Comment (FluffySheep) [inline src/main/java/com/google/codeu/servlets/StatsPageServlet.java]: nit (nitpicky): Remove this extra line under the doGet().

File: src/main/webapp/stats.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Stats</title>
+  <link rel="stylesheet" href="/css/main.css">
+  
+  <script>
+    // Fetch stats and display them in the page.
+    function fetchStats(){
```
Comment (FluffySheep) [inline src/main/webapp/stats.html]: Run all the javascript through https://prettier.io/playground/ (just copy and paste)

File: src/main/webapp/stats.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```
Comment (FluffySheep) [inline src/main/webapp/stats.html]: Add a screenshot to your PR (usually only add it if you edit html). Also rename your PR to have a meaningful title. Here is an example that Kendall wrote previously https://github.com/FluffySheep/CodeU2019-Team28/pull/27

**Comment by linkaren** [comment]
screen shot 2019-02-27 at 11 29 33 pm

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 5]
```diff
@@ -30,6 +31,8 @@
 /** Provides access to the data stored in Datastore. */
 public class Datastore {
 
+  static final int NUMBER = 1000;
```
Comment (FluffySheep) [inline src/main/java/com/google/codeu/data/Datastore.java]: Sorry could you rename this variable to MESSAGE_LIMIT and write a comment.

/** The maximum number of message entries to fetch from Datastore. */
static final int MESSAGE_LIMIT = 1000;

**Pull request merged**